src/Backups/IBackupCoordination.h

```diff
@@ -22,6 +22,7 @@ class IBackupCoordination
 
     /// Sets the current stage and waits for other hosts to come to this stage too.
     virtual void setStage(const String & new_stage, const String & message) = 0;
+    virtual void setStageForCluster(const String & new_stage) = 0;
```

Comment:
I think we don't need this extra function because there are only three stages which should be `for cluster`: "error", "completed" and "sheduled for start". I mean `BackupCoordinationStageSync` can understand by itself where to store a passed stage.

Reply:
We could also rewrite `waitForStage()` to make it subscribe for the value of the root node instead of subscribing for children of the `stage` node. Just to simplify the code.

Reply:
Sorry I didnt get how we can change `waitForStage()`, as this is used for the per node queries, so we will need to check the stage in the alive node.